charts/osm/templates/mutatingwebhook.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: admissionregistration.k8s.io/v1beta1
 kind: MutatingWebhookConfiguration
 metadata:
   labels:
-    app: osm-controller
+    app: osm-injector
```

Comment:
@shashankram Thinking more about this, it seems to me that actually moving to something more generic, like `osm` instead of `osm-controller` and `osm-injector` would be best.

Here is why....
I wanted to see what Kubernetes resources were created with `osm install`.

Before this PR I can use `kubectl get all -A --selector app=osm-controller`

Output is something like this:
```
$ kubectl get all -A --selector app=osm-controller

NAMESPACE    NAME                                  READY   STATUS    RESTARTS   AGE
osm-system   pod/osm-controller-545cbd8b68-98hgs   1/1     Running   0          5m22s

NAMESPACE    NAME                           TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)              AGE
osm-system   service/osm-config-validator   ClusterIP   11.1.111.111   <none>        9093/TCP             5m22s
osm-system   service/osm-controller         ClusterIP   11.0.111.112   <none>        15128/TCP,9092/TCP   5m22s

NAMESPACE    NAME                             READY   UP-TO-DATE   AVAILABLE   AGE
osm-system   deployment.apps/osm-controller   1/1     1            1           5m22s

NAMESPACE    NAME                                        DESIRED   CURRENT   READY   AGE
osm-system   replicaset.apps/osm-controller-545cbd8b68   1         1         1       5m22s
```

Sadly the osm-config ConfigMap does not appear on that list.

I thin that we should create one label, for all assets, which will identify things that were deployed by `osm install`.

Will open an issue with this.

Reply:
Hey @draychev I get what you mean. Though I think we could simply add an additional label for this purpose, ex. `kind: osm`.

I don't think labeling the app as `osm` for all resources is the correct thing to do here.